spec/models/report/standard_form_report_spec.rb

```diff
@@ -77,7 +77,9 @@
 
     it "should return non-submitting observers" do
       # make observers
-      observers = %w(bob jojo cass sal).map{|n| create(:user, :login => n, :role_name => :observer)}
+      observers = %w(bob jojo cass sal).map{|n| create(:user, login: n,
+                                                              role_name: :observer,
+                                                              name: n.capitalize)}
```

Comment:
Please avoid this style of indenting. It gives me nightmares :P  Just please break the line and indent one extra level.